js/ui/text_box/ui.text_editor.mask.strategy.default.js

```diff
@@ -35,6 +38,19 @@ class DefaultMaskStrategy extends BaseMaskStrategy {
             this._handleBackspaceInput(event);
         }
 
+        if(IS_SAFARI) {
```

Comment:
We have a similar issue with autofill in Google Chrome on Windows as well - see https://js.devexpress.com/Demos/WidgetsGallery/Demo/Validation/Overview/jQuery/. 

Let's try to get rid of this flag and fix this issue for Google Chrome, too.